docs/userguide/analysis/peak_finding.py

```diff
@@ -468,7 +468,7 @@
     table_property = "  ".join(
         [f"{properties[key][i].m:>10.3f}" for i in range(len(peaks))]
     )
-    title = f"{key:>.16} ({properties[key][0].u:>.6K})"
```

Comment:
```python 
title = f"{key:>.16} ({properties[key][0].u:~P})"
```  
will be nicer 🙂

Reply:
Todo also for the first line

Reply:
print(f'{"peak_position (cm⁻¹)":>26}: {table_pos}')

I copied the unicode directly